src/App.tsx

```diff
@@ -1,61 +1,110 @@
 import './App.scss';
+import { useState } from 'react';
+import { getUserById } from './services/user';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
+  const [userId, setUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setHasTitleError(true);
+    setHasUserIdError(true);
+
+    if (!title || !userId) {
+      // eslint-disable-next-line no-useless-return
+      return;
+    }
```

Comment:
set errors separately
![image](https://github.com/mate-academy/react_add-todo-form/assets/55200530/e9d4defc-0437-4c77-a971-829bd0d323c0)

now error is only on select user, but error appears for both inputs